lottie/src/main/java/com/airbnb/lottie/LottieConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.airbnb.lottie;
+
+import android.content.Context;
+
+import com.airbnb.lottie.network.LottieNetworkFetcher;
+import com.airbnb.lottie.network.LottieNetworkCacheProvider;
+
+import java.io.File;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+/**
+ * Class for custom library configuration
+ */
+public class LottieConfig {
+
+  @NonNull final Context applicationContext;
+  @Nullable final LottieNetworkFetcher networkFetcher;
+  @Nullable final LottieNetworkCacheProvider cacheProvider;
+
+  public LottieConfig(@NonNull Context applicationContext, @Nullable LottieNetworkFetcher networkFetcher, @Nullable
```

Comment:
I think you meant for this to be private

Reply:
done